src/vit-servicing-station/doc/api/v0.yaml

```diff
@@ -207,7 +207,7 @@ paths:
 
   /api/v0/challenges/{id}/{voter_group_id}:
     get:
-      operationId: getChallenge
+      operationId: getChallengeByIdAndVoterGroupId
       summary: Get challenge by id
```

Comment:
Should we update the summary to `Get challenge by id and voter group id`?

Reply:
yes, sure, good catch